neo4j/log/void.go

```diff
@@ -33,3 +33,6 @@ func (l Void) Warnf(name, id string, msg string, args ...interface{}) {
 
 func (l Void) Debugf(name, id string, msg string, args ...interface{}) {
 }
+
+func (l Void) LogMessage(src MessageSource, id, msg string, args ...interface{}) {
```

Comment:
Leftover?

Reply:
I initially thought it would be good to reuse `Void` for the NOOP Bolt logger, but it turns out to be not really useful, so I can create a separate (default) NOOP implementation instead.